jib-core/src/main/java/com/google/cloud/tools/jib/event/JibEventType.java

```diff
@@ -21,7 +21,7 @@
 /** Holds references to all {@link JibEvent} types. */
 public class JibEventType<E extends JibEvent> {
```

Comment:
Why is this class necessary?

Reply:
It's not completely necessary, but it's there to centralize all the event types so that users can find all the available event types to add handlers for in one location.

Reply:
How does it do that?  Oh, I see:
```
// TODO: Add entries for all JibEvent types.
```
Can't that be handled by an IDE's _Type Hierarchy_ and Javadoc showing all subclasses of `JibEvent`?

Reply:
That's true - should this just be removed then? The benefit I can see from `JibEventType` is that users would not need to have imports for the various `JibEvent` classes when they add handlers.

Reply:
Are we expecting this to be enum-esque?